src/testutils/transaction.ts

```diff
@@ -16,40 +16,50 @@ export class TestTransaction implements ISignable, IVerifiable {
     options: number = 0;
 
     sender: string = "";
+    guardian: string = "";
+    guardianSignature: string = "";
     signature: string = "";
 
     constructor(init?: Partial<TestTransaction>) {
         Object.assign(this, init);
     }
 
-    serializeForSigning(signedBy: IAddress): Buffer {
-        let sender = signedBy.bech32();
+    serializeForSigning(): Buffer {
         let dataEncoded = this.data ? Buffer.from(this.data).toString("base64") : undefined;
+        let guardian = this.guardian ? this.guardian : undefined;
```

Comment:
you can change the variable declaration to all the variables that are not modified from let to const